src/Polly.Shared/Duration/LinearBackoff.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+
+namespace Polly.Duration
+{
+    /// <summary>
+    /// Generates sleep durations in an linear manner.
+    /// The formula used is: Duration = <see cref="Delay"/> + <see cref="Factor"/> x iteration x <see cref="Delay"/>.
+    /// For example: 2s, 4s, 6s, 8s...
+    /// </summary>
+    public sealed class LinearBackoff : ISleepDurationStrategy
+    {
+        /// <summary>
+        /// Whether the first retry will be immediate or not.
+        /// </summary>
+        public bool FastFirst { get; }
+
+        /// <summary>
+        /// The duration value for the first retry.
+        /// </summary>
+        public TimeSpan Delay { get; }
```

Comment:
Could call this `InitialDelay` (and the corresponding ctor param, `initialDelay`). I realise this changes the pattern (such that not all similar are called Delay), but it is more directly what it is?

Reply:
Agree. Should we make the same change on `ExponentialBackoff`?
It's a little more ambiguous there, since it is indeed the initial-delay, but it's also the factor in the ongoing calculation